ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/dynamic-container-configuration.xml

```diff
@@ -117,6 +117,54 @@
         </object-param>
       </init-params>
     </component-plugin>
+    <!-- Add attachment app to the Content edition page -->
```

Comment:
what about note editor ?

Reply:
Attachment app already injected for notes in this file!